packages/vanilla/src/components/tabs.ts

```
@@ -1,102 +1,53 @@
-    'padding-left: 16px;' +
-    'width: 4.5rem !important;' +
-    'background: linear-gradient(90deg, rgba(256,256,256, 0) -25%, white 30%);\n' +
-    'background-attachment: local, local, scroll, scroll;'
-
-  const styledTabs = `
-    .tabs {
-        padding-right: 2rem;
-    }
-`
+export const initTabs = (): void => {
+  const tabList = document.getElementById('tablist') as HTMLDivElement;
```

ESLint

[error] Extra semicolon.

(semi)